.env.example

```diff
@@ -24,6 +24,9 @@ export CTR_REGISTRY_PASSWORD=
 # Default: osm-system
 export K8S_NAMESPACE=osm-system
 
+# optional: name of Kubernetes contexts that OSM will be deployed in multicluster mode
+# export MULTICLUSTER_CONTEXTS=()
```

Comment:
```suggestion
# export MULTICLUSTER_CONTEXTS=
```

Reply:
I want to indicate that this by default is an empty list. It will be used in for loop below